jsettlers.mapcreator/src/jsettlers/mapcreator/control/UndoRedoHandler.java

```diff
@@ -0,0 +1,131 @@
+package jsettlers.mapcreator.control;
+
+import java.util.LinkedList;
+
+import jsettlers.mapcreator.data.MapData;
+import jsettlers.mapcreator.data.MapDataDelta;
+import jsettlers.mapcreator.main.window.EditorFrame;
+
+/**
+ * Handle Undo / Redo actions
+ * 
+ * @author Andreas Butti
+ *
+ */
+public class UndoRedoHandler {
+
+	/**
+	 * Max undo counts
+	 */
+	private static final int MAX_UNDO = 100;
```

Comment:
Do we need such a low UNDO max? Is it an issue to have a much higher max? Could we even have an infinite or close to infinite max (without significant performance hits)?